src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -10267,5 +10206,118 @@ public class Class1
                 Assert.False(state.IsAvailable);
             }
         }
+
+        [Fact]
+        [WorkItem(15532, "https://github.com/dotnet/roslyn/issues/15532")]
+        public async Task ExtractLocalFunctionCall()
+        {
+            var code = @"
+class C
+{
+    public static void Main()
+    {
+        void Local() { }
+        [|Local();|]
+    }
+}";
+            await TestExtractMethodAsync(code, @"
+class C
+{
+    public static void Main()
+    {
+        void Local() { }
+        NewMethod();
+    }
+
+    private static void NewMethod()
+    {
+        Local();
+    }
+}");
```

Comment:
wait... this should already be supported.  Look at ExtractMethodTests.TestIsPattern and see how we do the following:

```c#
class C
{
    static void M(int i)
    {
        int r;
        int y;
        {|Rename:NewMethod|}(i, out r, out y);
        System.Console.WriteLine(r + y);
    }

    private static void NewMethod(int i, out int r, out int y)
    {
        r = M1(3 is int {|Conflict:y|}, i);
    }
}
```